package.json

```diff
@@ -69,6 +69,7 @@
     "axe-webdriverjs": "0.4.0"
   },
   "dependencies": {
+    "@okta/courage": "1.14.0",
```

Comment:
I think so, but if the developer actually clone the repository, npm install would be good as well since we have put one under `node_modules`, I don't know why `optionalDependencies` though.